pkg/pillar/hypervisor/hypervisor.go

```diff
@@ -18,6 +19,12 @@ import (
 var currentHypervisor Hypervisor
 
 func init() {
+	if fileutils.DirExists(nil, "/opt/pillar/opt") {
```

Comment:
other things I tried without success:
* /media/boot
* touch /tmp/installation in install script
* touch /var/tmp/installation in install script

Reply:
The installer creates and writes at least one line to /run/installer.log
I suspect that will be more stable than looking at the directory you looked for.
And it makes sense to add a comment to installer that there is a dependency on using that file.

Alternatively we can add a --install option to zedbox which we will use in installer's pillar_run.